### PR TITLE
added DynamicContactCenter project

### PR DESCRIPTION
**Pull request by swalmsley11**
*Issue #, if available:*

*Description of changes:*

- Adding Dynamic Contact Center project under `projects/DynamicContactCenter`
- Updated `README.md` with project details
- Updated `projects/README.md` with project details

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/DynamicAttributeRouting/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  * Associate all the queues to an existing routing profile
+  ![routing_profile](Images/routing_profile.png)
+
+## Deployment Steps
+Perform the following steps to deploy this project: 
+1. Create a new CloudFormation stack and upload template `dynamic-attribute-routing.yaml`
+  * Make sure that you select the correct region to match your Amazon Connect instance
+2. Name your stack and enter all the parameters listed with details from your Amazon Connect instance
+  * Take note of the `DeploymentEnvironment` you select
+  * Make sure you enter enter the correct phone number mapping and queue ARNs you noted previously
```
Comment (angieyu) [inline projects/DynamicAttributeRouting/README.md]: enter is used twice - remove one

File: projects/DynamicAttributeRouting/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+3. Click `Next`, select the IAM resource disclaimer and click `Create Stack`
+4. Once the stack creation has completed, add the `~GetLambdas`, `~GetDynAttributes`, and `~GetPrompts` Lambda functions to your Amazon Connect instance.
+  ![associate_lambdas](Images/associate_lambdas.png)
+5. Download the three contact flows in the ContactFlows directory.
+6. Import them in the order listed below. Save and Publish each contact flow after import while making the edits as indicated below:
+  * (MAIN FLOW) `100_MainFlow` - Publish as-is
+  * (LANGUAGE FLOW) `100_LangFlow` - Publish as-is
+  * (INIT FLOW) `100_InitFlow` - Select the first Invoke AWS Lambda Function block and change it to the GetLambdasFunction that you enabled previously (in step 2). In the same block, set the value for the `environment` key to the same environment as the CloudFormation stack.
+  ![invoke_lambda](Images/invoke_lambda_block.png)
+7. Open your `Default customer queue` contact flow and edit the Loop Prompt block. Remove the default queue hold message. Keeping this message will create is disjointed caller experience for non-english calls.
```
Comment (angieyu) [inline projects/DynamicAttributeRouting/README.md]: will create "a" disjointed, instead of "is"?

Comment (angieyu) [reply projects/DynamicAttributeRouting/README.md]: Also, capitalize English

File: projects/DynamicAttributeRouting/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  * Don't forget to `Save` and `Publish` the flow after making changes
+  ![loop_prompts](Images/loop_prompts.png)
+8. Set the set the `100_InitFlow` as the contact flow for all three phone numbers in your instance.
```
Comment (angieyu) [inline projects/DynamicAttributeRouting/README.md]: Set the set?

File: projects/DynamicAttributeRouting/README.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  ![dialed_numbers-2](Images/dialed_numbers-2.png)
+9. Wait ~ 2 min for everything to update.
```
Comment (angieyu) [inline projects/DynamicAttributeRouting/README.md]: ~2min remove space

File: projects/DynamicAttributeRouting/README.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+## Validation
+To validate everything is working as expected, perform the following steps:
```
Comment (angieyu) [inline projects/DynamicAttributeRouting/README.md]: to validate "that" everything

File: projects/DynamicAttributeRouting/README.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+1. Call the first phone number that you assigned for English
+2. Listen to the prompts and verify they play correctly
+3. View the real-time metrics and verify the call was placed in the correct queue
+4. Hang up. 
+
+### Test additional languages 
+Follow the same steps listed above to test additional languages. Each number should playback the appropriate language-specific prompts and place the call in the correct queue.
+
+## Conclusion
+Now that you have the foundation in place for attribute-based dynamic routing the possibilites are endless. Utilizing session attributes allows you to keep your CTR records clean and only write the attributes you want while keeping the others in memory to perform various tasks throughout the contact.
```
Comment (angieyu) [inline projects/DynamicAttributeRouting/README.md]: dynamic routing, the possibilities  - add comma, fix spelling for "possibilities"

File: projects/DynamicAttributeRouting/README.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+Alpha to Zulu Corp distributes widgets across North America and has customers in the US, Canada and Mexico. They provide language specific numbers for customers to contact them and speak with a live agent. Each phone number routes to a specific language queue while playing prompts in that language
+
+## Project Components
+* CloudFormation template that builds:
+  * IAM Role: Allows Lambda read/write access to Dynamo DB
+  * Lambda Functions
+    * Retrieve dynamic attributes from various tables
+    * Init function to populate DynamoDB tables
+  * DynamoDB Tables
+    * Stores dynamic attributes for lambda functions, dialed numbers and prompts
```
Comment (angieyu) [inline projects/DynamicAttributeRouting/README.md]: Capitalize Lambda

**Review by angieyu — COMMENTED** [review]
Found spelling error, please address

File: projects/DynamicAttributeRouting/README.md (hunk 1 above)
Comment (swalmsley11) [reply projects/DynamicAttributeRouting/README.md]: fixed

File: projects/DynamicAttributeRouting/README.md (hunk 2 above)
Comment (swalmsley11) [reply projects/DynamicAttributeRouting/README.md]: fixed

File: projects/DynamicAttributeRouting/README.md (hunk 3 above)
Comment (swalmsley11) [reply projects/DynamicAttributeRouting/README.md]: fixed

File: projects/DynamicAttributeRouting/README.md (hunk 4 above)
Comment (swalmsley11) [reply projects/DynamicAttributeRouting/README.md]: fixed

File: projects/DynamicAttributeRouting/README.md (hunk 5 above)
Comment (swalmsley11) [reply projects/DynamicAttributeRouting/README.md]: fixed

File: projects/DynamicAttributeRouting/README.md (hunk 6 above)
Comment (swalmsley11) [reply projects/DynamicAttributeRouting/README.md]: fixed

File: projects/DynamicAttributeRouting/README.md (hunk 7 above)
Comment (swalmsley11) [reply projects/DynamicAttributeRouting/README.md]: fixed

**Comment by angieyu** [comment]
approved

**Pull request merged**